src/main/java/de/hysky/skyblocker/skyblock/chocolatefactory/ChocolateFactorySolver.java

```diff
@@ -30,6 +31,8 @@ public class ChocolateFactorySolver extends ContainerSolver {
 	private static final Pattern CPS_PATTERN = Pattern.compile("([\\d,.]+) Chocolate per second");
 	private static final Pattern CPS_INCREASE_PATTERN = Pattern.compile("\\+([\\d,]+) Chocolate per second");
 	private static final Pattern COST_PATTERN = Pattern.compile("Cost ([\\d,]+) Chocolate");
+	private static final Pattern LEVEL_PATTERN = Pattern.compile("\\[(\\d+)]");
+	private static final Pattern COACH_LEVEL_PATTERN = Pattern.compile("Coach Jackrabbit (\\w+) ");
```

Comment:
Did you mean to have a space at the end?
```suggestion
	private static final Pattern COACH_LEVEL_PATTERN = Pattern.compile("Coach Jackrabbit (\\w+)");
```